pms/models/pms_sale_channel.py

```diff
@@ -0,0 +1,12 @@
+from odoo import fields, models
+
+
+class PmsSaleChannel(models.Model):
+    _name = "pms.sale.channel"
+    _description = "Sales Channel"
+
+    # Fields declaration
+    name = fields.Text(string="Sale Channel Name")
```

Comment:
Better to use String instead of Text

Reply:
And use attribute translate=True on all fields that you think it makes sense to use in different languages by the user lang ("Door" - "Puerta", etc...)